packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/StockDistribution.tsx

```diff
@@ -106,13 +112,20 @@ const MonthlyBar = ({
         borderColor: 'gray.dark',
         borderStyle: 'solid',
         height: '20px',
+        overflow: 'hidden',
```

Comment:
on the items where I've added far too much stock, the monthly summary is showing so very small we can't see anything! I've added the overflow to make it look ok, and a tooltip to show the details.